static/app/index.tsx

```diff
@@ -1,10 +1,13 @@
 async function app() {
-  const [{bootstrap}, {initializeMain}] = await Promise.all([
-    import('app/bootstrap'),
-    import('app/bootstrap/initializeMain'),
-  ]);
-  const data = await bootstrap();
-  initializeMain(data);
+  // We won't need initalizeMain until we've completedd bootstrap. We can start
+  // fetching it now, but don't await on it.
+  const initMain = import('app/bootstrap/initializeMain');
+  const {bootstrap} = await import('app/bootstrap');
+
+  const config = await bootstrap();
+  const {initializeMain} = await initMain;
+
+  initializeMain(config);
```

Comment:
I think this is slightly harder to read, I would make it a bit more verbose and have some intermediate vars that hold the promises for the `import()`


```javascript
  const [bootstrapPromise, initializeMainPromise] = [
    import('app/bootstrap'),
    import('app/bootstrap/initializeMain'),
  ];
  const {bootstrap} = await bootstrapPromise;
  const config = await bootstrap();
  ...
```

Reply:
good call, I like it